forwards_paper.tex

```diff
@@ -332,12 +348,16 @@ \subsection*{Recording the pedigree in forwards time}
 \algstep{W5.}{New node.}{Set $P'_j \leftarrow n$ and $\tau_n \leftarrow t$.
 }
 
-\algstep{W6.}{Choose parents.}{Set $a \leftarrow \randomuniform(\{0, \dots, N - 1\})$,
-    $b \leftarrow \randomuniform(\{0, \dots, N - 1\})$ and $x \leftarrow \randomuniform([0, L))$.
+\algstep{W6.}{Choose parents.}{Set $a \leftarrow \randomuniform(\{0, \dots, n - 1\})$,
```

Comment:
That should be N, not n, right?

Reply:
Oops yeah. This was related to my confusion about the P_a indices. I'll
look at your edits and confirm it's cleared up